spec/types/jtd-schema.spec.ts

```diff
@@ -0,0 +1,212 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
+import type {JTDSchema} from "../.."
+import chai from "../chai"
+const should = chai.should()
+
+describe("JTDSchema typechecks", () => {
+  it("should typecheck number schemas", () => {
+    const numf: JTDSchema<number> = {type: "float64"}
+    should.exist(numf)
```

Comment:
if the only reason why should.exist is needed is to prevent typescript complaining could we prefix with underscores and remove all should.exist to reduce the noise?

Reply:
Both eslint and typescript complained, and I wasn't sure of a good solution. If both are set to ignore underscores, then yeah, that makes much more sense.

Reply:
eslant is definitely set to ignore names that start from underscore, I "think" typescript would ignore them too.